allure-generator/src/main/javascript/helpers/attachment-uri.js

```diff
@@ -0,0 +1,11 @@
+import {SafeString} from 'handlebars/runtime';
+
+export default function attachmentUri(source) {
+  let sourceUri;
+  if (source.startsWith('http')) {
+    sourceUri = source;
+  } else {
+    sourceUri = 'data/attachments/' + source;
+  }
+  return new SafeString(`<a class="link" href="${sourceUri}" target="_blank" data-tooltip="Open attachment in new tab">`);
```

Comment:
Also, the helper should be responsible only for URL generation, not the `<a>` part. Just return `sourceUri` out here and leave opening `<a>` at the place where it was before

Reply:
I will make these changes in the new PR